packages/client/src/components/form/FetchButton.tsx

```diff
@@ -13,7 +13,7 @@ import styled from 'styled-components'
 import { ApolloError, ApolloQueryResult, ApolloConsumer } from '@apollo/client'
```

Comment:
All of the changes in the `client` are just removing the dependency link between client and gateway. This was necessary to get client compiling without having to add a support for `commons` package in client. We should get rid of these types altogether and start using the types generated from both the client graphql queries and gateway schema.